application/backend/src/main/resources/apps/typerefinery/components/widgets/map/dialog/.content.json

```diff
@@ -1,15 +1,103 @@
 {
   "sling:resourceType": "wcm/dialogs/dialog",
   "tabs": {
-      "sling:resourceType": "wcm/dialogs/components/tabs",
-      "generalTab": {
-          "sling:resourceType": "wcm/dialogs/components/tab",
-          "label": "General",
-          "title": {
-              "sling:resourceType": "wcm/dialogs/components/textfield",
-              "name": "title",
-              "label": "Title"
-          }
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "configTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "lat": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "mapLat",
+        "label": "Latitude",
+        "description": "These will be added as map latitude"
+      },
+      "long": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "mapLng",
+        "label": "Longitude",
+        "description": "These will be added as map longitude"
+      },
+      "zoomLevel": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "zoomLevel",
+        "label": "Zoom Level",
+        "description": "These will be added as map level"
       }
+    },
+    "tilesTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "Tiles layer",
+      "tileTemplate": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "tileTemplate",
+        "label": "Tile Template",
+        "description": "Instantiates a tile layer object given a URL template "
+      },
+      "layerZoom": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "layerZoom",
+        "label": "Layer Zoom",
+        "description": "The minimum zoom level down to which this layer will be displayed (inclusive)."
+      },
+      "copyRightUrl": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "copyRightUrl",
+        "label": "Copyright Url",
+        "description": "CopyRight Url"
+      }
+    },
+    "markerTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "Marker",
+      "markerList": {
+        "name": "markerList",
+        "label": "Marker Details",
+        "sling:resourceType": "wcm/dialogs/components/multifield",
+        "marker-lat": {
```

Comment:
Please follow the camel casing standard.

In most cases, the key and name attributes should be the same. 